feature-libs/order/components/order-details/order-consignments.service.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { OrderEntry } from '@spartacus/cart/base/root';
+import { Order, Consignment, ReplenishmentOrder } from '@spartacus/order/root';
+import {
+  completedValues,
+  cancelledValues,
+} from './order-detail-items/order-consigned-entries/order-consigned-entries.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OrderConsignmentsService {
```

Comment:
[QUESTION]
For consistency, shall we rename it to `MyAccountV2OrderConsignmentsService`, I guess?

Reply:
done